Dockerfile

```diff
@@ -2,7 +2,11 @@ FROM python:3.10.2-slim-buster
 
 WORKDIR /app
 
-RUN apt-get update && apt-get install -y ffmpeg git
+ENV PYTHONDONTWRITEBYTECODE 1
+
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update && apt-get install -y ffmpeg git gcc
```

Comment:
do we need gcc?

Reply:
one of the python packages couldn't install without `gcc` in my environment